### ISSUE TITLE
"Incorrect Phone Number" Error Message still showing even if format is correct. 

### ISSUE THREAD

**Issue by ac617** [BenediktBergmann/PCF-Controls#100]
Hi Benedikt,

Thank you for creating the CheckPhoneNumber control PCF control.
We are currently using it on the Lead.mobilephone field.
The error message works fine if we manually enter the mobilephone number on the form and save the record.

However, we have a scenario where we populate lead.parencontactid field which then autopopulates the lead.mobilephone field from the contact record (MSFT out the box behavior).

![image](https://github.com/BenediktBergmann/PCF-Controls/assets/138644494/8e1b9446-06da-4852-a549-d93388708935)

The only way to remove the error message is to save the record.
![image](https://github.com/BenediktBergmann/PCF-Controls/assets/138644494/7993640a-441d-4a49-918f-58047df7df05)

Is there a way to correct the error message onchange of the lead.mobilephonefield when lead.parentcontactid is populated?

**Comment by BenediktBergmann** [comment]
Thanks for reporting this.
I will take a look at it.

**Comment by BenediktBergmann** [comment]
@ac617 I found a bug in the function which checks whether the phone number is valid. Instead of using the value sent in it would get the field's value. In your case, the field still has a value of "---" which is not a correct number.
I fixed this with my last commit. This fix will be included in the next version of my PCFs-
I plan on creating a new release sometime next week. There is another Issue with another PCF I would like to fix before creating a new release.
